Tests/PointFreeTests/AccountTests/AccountTests.swift

```diff
@@ -309,7 +288,11 @@ final class AccountTests: TestCase {
   }
 
   func testAccountCanceledSubscription() {
-    update(&Current, \.stripe.fetchSubscription .~ const(pure(.canceled)))
+    update(
+      &Current,
+      \.stripe.fetchSubscription .~ const(pure(.canceled)),
+      \.database.fetchSubscriptionById .~ const(pure(.canceled))
```

Comment:
ive had an old Trello card to fix this test and I finally am. basically this snapshot would render with the "private RSS feed" link on the account page, even tho the subscription had been canceled. I wasn't mocking the right thing.